packages/vscode-extension/src/commonlib/log.ts

```diff
@@ -14,7 +14,10 @@ export class VsCodeLogProvider implements LogProvider {
   private static instance: VsCodeLogProvider;
 
   private constructor() {
-    this.outputChannel = vscode.window.createOutputChannel(outputChannelDisplayName);
+    this.outputChannel = vscode.window.createOutputChannel(
+      outputChannelDisplayName,
+      "teamsfx-toolkit-output"
```

Comment:
Is this a reusable constant or just to align with `package.json`?
If reusable, better to define in some `constants.ts` file.
If just to align with `package.json`, add comment such as *// Align with languages:id value in package.json*, for better maintenance.

Reply:
Thank you! The comment is added.